sdk/include/psdk/ntstatus.h

```diff
@@ -234,6 +235,8 @@ extern "C" {
 #define STATUS_CLUSTER_NETWORK_ALREADY_OFFLINE  ((NTSTATUS)0x80130004)
 #define STATUS_CLUSTER_NODE_ALREADY_MEMBER      ((NTSTATUS)0x80130005)
 
+#define STATUS_FLT_BUFFER_TOO_SMALL             ((NTSTATUS)0x801C0001L)
```

Comment:
Why the empty line at line237? Is there anything that would separate those 2 kind of STATUS_ defines into different classes? (Same question for the line 239)

Reply:
Obviously. Above you have some status codes for the cluster component, then that one for the FS filter component.

Reply:
I see empty line was used to separate different kind of STATUS, so do I.
BTW I noticed `L` should be removed to make it tidy, I'll commit it later.